indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamSequenceNumbers.java

```diff
@@ -32,7 +32,7 @@
     @Type(name = "start", value = SeekableStreamStartSequenceNumbers.class),
     @Type(name = "end", value = SeekableStreamEndSequenceNumbers.class)
 })
-public interface SeekableStreamSequenceNumbers<PartitionIdType, SequenceOffsetType>
+public interface SeekableStreamSequenceNumbers<PartitionIdType, SequenceOffsetType> extends Comparable<SeekableStreamSequenceNumbers<PartitionIdType, SequenceOffsetType>>
```

Comment:
can we add some Javadocs to explain what Comparable means in this context and how it applies to sequence offsets?